Author: aszlig <aszlig@nix.build>
Date:   Sat Mar 31 00:31:11 2018 +0200

    Remove instagram link from news
    
    We do have a canary for Instagram and also of course don't want to link
    to external resources too much, so let's remove that link in the news.
    
    Aside from Instagram, the news entry also contained links to Facebook
    and others, which I removed as well.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: news-remove-instagram.patch

diff --git a/website/server/controllers/api-v3/news.js b/website/server/controllers/api-v3/news.js
index 569f62f6f..1c0df5184 100644
--- a/website/server/controllers/api-v3/news.js
+++ b/website/server/controllers/api-v3/news.js
@@ -3,7 +3,7 @@ import { authWithHeaders } from '../../middlewares/auth';
 let api = {};
 
 // @TODO export this const, cannot export it from here because only routes are exported from controllers
-const LAST_ANNOUNCEMENT_TITLE = 'HABITICA ON INSTAGRAM, SELF-IMPOSED CHALLENGE BLOG, AND WHAT\'S THE APRIL FOOL UP TO?';
+const LAST_ANNOUNCEMENT_TITLE = 'SELF-IMPOSED CHALLENGE BLOG, AND WHAT\'S THE APRIL FOOL UP TO?';
 const worldDmg = { // @TODO
   bailey: false,
 };
@@ -34,13 +34,11 @@ api.getNews = {
         </div>
         <h2>3/30/2018 - ${LAST_ANNOUNCEMENT_TITLE}</h2>
         <hr/>
-        <h3>Habitica is Now on Instagram!</h3>
-        <p>Habitica's <a href='https://www.instagram.com/habitica' target='_blank'>Instagram</a> is now live! Follow us for updates and exclusive peeks at behind-the-scenes fun!</p>
         <div class="media align-items-center">
           <div class="scene_todos"></div>
           <div class="media-body">
             <h3>New Blog Post: Self-Imposed Challenges!</h3>
-            <p>This month's <a href='https://habitica.wordpress.com/2018/03/28/self-imposed-challenges/' target='_blank'>featured Wiki article</a> is about Self-Imposed Challenges! We hope that it will help you as you customize Habitica to make it as fun and challenging as you'd like it to be! Be sure to check it out, and let us know what you think by reaching out on <a href='https://twitter.com/habitica' target='_blank'>Twitter</a>, <a href='http://blog.habitrpg.com' target='_blank'>Tumblr</a>, and <a href='https://facebook.com/habitica' target='_blank'>Facebook</a>.</p>
+            <p>This month's <a href='https://habitica.wordpress.com/2018/03/28/self-imposed-challenges/' target='_blank'>featured Wiki article</a> is about Self-Imposed Challenges! We hope that it will help you as you customize Habitica to make it as fun and challenging as you'd like it to be! Be sure to check it out.</p>
             <div class="small mb-3">by Beffymaroo and the Wiki Wizards</div>
           </div>
         </div>
